types/solc/solc-tests.ts

```diff
@@ -0,0 +1,6 @@
+import * as solc from "solc";
+
+solc.loadRemoteVersion(
+    "1.0.0",
+    (err: Error | null, solcVersion?: solc.SolcInstance) => {}
+);
```

Comment:
Please add more tests from readme
eg. https://github.com/ethereum/solc-js#from-version-0411
https://github.com/ethereum/solc-js#linking-bytecode
https://github.com/ethereum/solc-js#updating-the-abi
https://github.com/ethereum/solc-js#formatting-old-json-assembly-output